script/rsync.sh

```diff
@@ -36,7 +36,7 @@ set +e
     [ -e "$subfolder/print_openqa.sh" ] || { >&2 echo "No file found: {$subfolder/print_openqa.sh}"; exit 1; }
 
     # nowhere to log yet as we haven't created $logfolder
-    bash -e "$subfolder/read_files.sh" ||  { >&2 echo "read_files.sh failed for $subfolder in enviroment $environ"; exit 1; }
+    bash -e "$subfolder/read_files.sh" || { >&2 echo "read_files.sh failed for $subfolder in enviroment $environ"; exit 0; }
```

Comment:
If we do this then we should be more explicit about the impact, i.e. at least rewording the text string to make it clear that the error is basically ignored. But maybe that would be better to decide within read_files.sh and not here in a general way.